build/build.xml

```diff
@@ -628,6 +628,9 @@
       <!--<fileset dir="shared" includes="launch4j/**" />-->
       <fileset dir="shared" includes="lib/**" excludes="lib/fonts/**" />
       <fileset file="shared/revisions.txt" />
+
```

Comment:
I have added two spaces here to avoid merge conflicts with my other pending Pull Requests which modifies this same file.